scripts/generate.sh

```diff
@@ -0,0 +1,19 @@
+# Get latest OpenAPI specification
+git clone https://gitlab.zalopay.vn/accounting/dojo/docusaurus-playground.git
```

Comment:
@noradomi should remove this internal domain?

Reply:
@anhldbk okie anh. vì hiện tại chưa có chỗ public host open API spec nên e đang pull từ đây. Chắc phải để chung spec với sdk

Reply:
@qtdan @noradomi consider of having a dedicated repository for our OpenAPI specifications?